### PR TITLE
Add form upload component and logout

### LINKED ISSUE #4: Create file upload form
Blocked by #3 

Create the frontend components necessary for a form upload

### PR DESCRIPTION
**Pull request by cailyncodes**
Should close #4 and #17

### CODE REVIEW

File: client/src/containers/Header/Header.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+const Header: React.FC = () => {
+	const { loading, user } = useContext(UserContext);
+
+	if (loading) {
+		return <></>;
+	}
+
+	return (
+		<header>
+			<h1>OEC Data Collection</h1>
```
Comment (mmazanec22) [inline client/src/containers/Header/Header.tsx:14]: I know this is a common pattern, but according to Deque it's a better practice for navigability to have the `h1` tag for the page be at the start of the `main` content: https://dequeuniversity.com/assets/html/jquery-summit/html5/slides/headings.html

File: client/src/containers/Upload/Upload.tsx [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+		const file = e.target.files[0];
+		formData.delete("file");
+		formData.set("file", file);
+		return false;
+	}
+	return (
+		<>
+			<p>{status && status.message}</p>
+			<form onSubmit={onSubmit}>
+				<label htmlFor="file"></label>
```
Comment (mmazanec22) [inline client/src/containers/Upload/Upload.tsx:43]: htmlFor needs to refer to an ID-- so we should add `id="file"` on the input

File: client/src/containers/Upload/Upload.tsx [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+			return;
+		}
+		const file = e.target.files[0];
+		formData.delete("file");
+		formData.set("file", file);
+		return false;
+	}
+	return (
+		<>
+			<p>{status && status.message}</p>
```
Comment (mmazanec22) [inline client/src/containers/Upload/Upload.tsx:41]: If this is an error message, we should use `aria-describedby` to associate it with the input field-- so the `p` would have an id, and the input would have aira-describedby that id.  We should also add `aria-invalid="true"` to the input field if there's an error (... and I should go make sure we're doing that in ECE reporter!).

We also need to send the focus to the input field if there's an error.

File: client/src/containers/Upload/Upload.tsx [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+			},
+			body: formData
+		})
+		.then(value => value.json())
+		.then(value => setStatus(value))
+		.catch(_ => setStatus({
+			error: true,
+			message: 'There was an error'
+		}));
+		e.persist();
```
Comment (jlhogan) [inline client/src/containers/Upload/Upload.tsx:25]: what does this do ?

Comment (cailyncodes) [reply client/src/containers/Upload/Upload.tsx:25]: Oh that was just a debugging thing

File: client/src/containers/Upload/Upload.tsx (hunk 3 above)
Comment (cailyncodes) [reply client/src/containers/Upload/Upload.tsx:41]: This is going to need to be rewritten once we get the component library in, so going to punt on these changes for now. But 100% will need to address them before releasing any version of this.

**Pull request merged**